setupaliases.lic

```diff
@@ -22,6 +22,9 @@
   # echos the full data of the room you're standing in
   ['lr', "#{$clean_lich_char}e echo Room[Room.current.id].inspect"],
 
+  # echos the full data of a specified remote room
+  ['lfr', "#{$clean_lich_char}e echo Room[\\?].inspect"],
```

Comment:
I expected this to be `\?` not `\\?` - did you run this to be sure?

Reply:
I believe its to escape the first slash so ;alias can insert the argument there. See the `fz` command

Reply:
`fz` was broken until sarvatt added the second slash

Reply:
yeah I ran it first to be sure, had to fix up a few others not working without the extra escape

Reply:
I added the `fz` command manually via ;alias then assumed it would work with ;setupaliases without any changes. Good fix 👍